users/permissions.py

```diff
@@ -11,13 +11,16 @@ def has_object_permission(self, request, view, obj):
 class IsAdminUser(permissions.BasePermission):
     message = 'You are not allowed to enter - for admin only.'
     def has_permission(self, request, view):
-        return request.user and request.user.is_authenticated and request.user.user_type == CustomUser.UserType.ADMIN.name
+        return request.user and request.user.is_authenticated \
+        and request.user.user_type == CustomUser.UserType.ADMIN.name
```

Comment:
If you have long statement to check it is better to divide it into variables. For examples like this:
```python
    def has_permission(self, request, view):
        is_user_authenticated = request.user and request.user.is_authenticated
        is_user_admin = request.user.user_type == CustomUser.UserType.ADMIN.name
        return  is_user_authenticated and is_user_admin
```

Reply:
Done